hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -240,7 +240,7 @@ public Record putOrUpdateReplicatedRecord(Data dataKey, Record replicatedRecord,
     public void removeReplicatedRecord(Data dataKey) {
```

Comment:
```suggestion
    public void removeReplicatedRecord(Data dataKey, boolean backup) {
```
better to have this flag in method signature here.

Reply:
Addressed in https://github.com/hazelcast/hazelcast/pull/22082/commits/163b1da58d9340fb68fc704cf1a861947c422d79